src/components/filter/filter.css

```diff
@@ -0,0 +1,33 @@
+.filter-container {
+  display: flex;
+  padding: 20px;
+  margin: 10px;
+  align-items: center;
+  justify-content: center;
+}
+.filter-container__list {
+  display: flex;
+  gap: 20px;
+  list-style: none;
+}
+.filter-container__list-item {
+  display: flex;
+  font-size: 14px;
+  border: 1px solid #dcdcdc;
+  border-radius: 16px;
+  padding: 5px 16px;
+  color: #717171;
+  cursor: pointer;
+}
+.filter-container__list-item:hover {
+  background-color: #f7f7f7;
+}
+.filter-container__list-text {
+  display: flex;
+  align-items: center;
+}
+.filter-container__image {
```

Comment:
you changed this class but changed in only one div, then, the layout now is not correct 🤔  

<img width="252" alt="image" src="https://user-images.githubusercontent.com/38358239/155131855-ee7e7659-b2e8-4f1d-85f6-e2a412b661ef.png">

<img width="604" alt="image" src="https://user-images.githubusercontent.com/38358239/155131918-4d1c1b6d-31d8-488e-b2e3-9f0670f7dc2c.png">

Reply:
Really 😞 

I changed the other classes:

<img width="1680" alt="Captura de Tela 2022-02-22 às 21 20 09" src="https://user-images.githubusercontent.com/42494406/155242042-b586a1f0-df0d-41ca-bf6b-fc3f0e28be93.png">